test/new-e2e/pkg/utils/e2e/client/agent_cmd_runner.go

```diff
@@ -81,8 +81,9 @@ func (agent *AgentCommandRunner) Hostname(commandArgs ...AgentArgsOption) string
 }
 
 // Config runs config command and returns the runtime agent config
-func (agent *AgentCommandRunner) Config(commandArgs ...AgentArgsOption) string {
-	return agent.executeCommand("config", commandArgs...)
+func (agent *AgentCommandRunner) Config(commandArgs ...AgentArgsOption) (string, error) {
```

Comment:
I actually removed relevant files when moving everything under `test/new-e2e/tests` .......

Added them back in aa4b8f67abdf8b9bcaeff687d41cf56527a88d21 sorry for that 🤦

Reply:
Did you consider having 2 functions, one returning an error and one that doesn't, because most of the time you don't want to handle the error when using config.

Reply:
Done in ae8ba37fce4a41d5a96b7e490991ca88e77b178e